Packages/MIES/MIES_Configuration.ipf

```diff
@@ -0,0 +1,1785 @@
+
+static StrConstant EXPCONFIG_FIELD_CTRLTYPE = "Type"
+static StrConstant EXPCONFIG_FIELD_CTRLVVALUE = "NumValue"
+static StrConstant EXPCONFIG_FIELD_CTRLSVALUE = "StrValue"
+static StrConstant EXPCONFIG_FIELD_CTRLSDF = "DataSource"
+static StrConstant EXPCONFIG_FIELD_CTRLDISABLED = "Disabled"
+static StrConstant EXPCONFIG_FIELD_CTRLPOSHEIGHT = "Height"
+static StrConstant EXPCONFIG_FIELD_CTRLPOSWIDTH = "Width"
+static StrConstant EXPCONFIG_FIELD_CTRLPOSTOP = "Top"
+static StrConstant EXPCONFIG_FIELD_CTRLPOSLEFT = "Left"
+static StrConstant EXPCONFIG_FIELD_CTRLPOSRIGHT = "Right"
+static StrConstant EXPCONFIG_FIELD_CTRLPOSPOS = "Pos"
+static StrConstant EXPCONFIG_FIELD_CTRLPOSALIGN = "Align"
+static StrConstant EXPCONFIG_FIELD_CTRLUSERDATA = "Userdata"
+static StrConstant EXPCONFIG_FIELD_BASE64PREFIX = "Base64 "
+static StrConstant EXPCONFIG_FIELD_CTRLARRAYVALUES = "Values"
+
+static StrConstant EXPCONFIG_UDATA_NICENAME = "Config_NiceName"
+static StrConstant EXPCONFIG_UDATA_JSONPATH = "Config_JSONPath"
+static StrConstant EXPCONFIG_UDATA_EXCLUDE_SAVE = "Config_DontSave"
+static StrConstant EXPCONFIG_UDATA_EXCLUDE_RESTORE = "Config_DontRestore"
+static StrConstant EXPCONFIG_UDATA_BUTTONPRESS = "Config_PushButtonOnRestore"
+// Lower means higher priority
+static StrConstant EXPCONFIG_UDATA_RESTORE_PRIORITY = "Config_RestorePriority"
+static StrConstant EXPCONFIG_UDATA_PANELTYPE = "Config_PanelType"
+static StrConstant EXPCONFIG_UDATA_WINHANDLE = "Config_WindowHandle"
+static StrConstant EXPCONFIG_UDATA_RADIOCOUPLING = "Config_RadioCouplingFunc"
+static StrConstant EXPCONFIG_UDATA_CTRLARRAY = "ControlArray"
+static StrConstant EXPCONFIG_UDATA_CTRLARRAYINDEX = "ControlArrayIndex"
+
+static Constant EXPCONFIG_UDATA_MAXCTRLARRAYINDEX = 100
+
+// DA_Ephys specific constants
+static StrConstant DAEPHYS_PANELTYPECODE = "DA_Ephys"
+static StrConstant DAEPHYS_UDATA_WINHANDLE = "DAEphys_WindowHandle"
+// Headstage checkboxes ctrl niceName prefix
+static StrConstant DAEPHYS_HEADSTAGECTRLARRAYPREFIX = "Check_DataAcqHS"
+static StrConstant DAEPHYS_EXCLUDE_CTRLTYPES = "12;9;10;4;"
+
+static StrConstant EXPCONFIG_DEFAULT_CTRL_JSONPATH = "Generic"
+static StrConstant EXPCONFIG_RESERVED_DATABLOCK = "GUI independent configuration data"
+
+static StrConstant EXPCONFIG_EXCLUDE_USERDATA = "ResizeControlsInfo;"
+static StrConstant EXPCONFIG_EXCLUDE_CTRLTYPES = "12;9;10;"
+
+/// @brief Creates a json with default experiment configuration block
+///
+/// @returns json with default experiment configuration
+static Function CONF_DefaultSettings()
+
+	variable jsonID
+
+	jsonID = JSON_New()
+
+	JSON_AddString(jsonID, AMP_TITLE, "0,1;2,3;4,5;6,7")
+	JSON_AddString(jsonID, AMP_SERIAL, "0;")
+	JSON_AddString(jsonID, PRESSURE_DEV, "Pressure1and2;Pressure3and4;Pressure5and6;Pressure7and8")
+	JSON_AddString(jsonID, PRESSURE_CONST, "0.4;0.14;0.05;0.14;0.29;0.03;0.05;0.045")
```

Comment:
@t-b default PRESSURE_CONST would probably be best set to zero for all headstages.

Reply:
ok

Reply:
@MichaelHuth I figured and should have caught it long before it reached you.